scripts/release/asset_publish.py

```diff
@@ -420,14 +431,14 @@ def update_asset_metadata(mlclient: MLClient, asset: AssetConfig):
             with open(spec_path) as f:
                 model_spec = YAML().load(f)
                 tags = model_spec.get("tags", {})
-                # convert tag value to string
-                for name, value in tags.items():
-                    if isinstance(value, dict):
-                        value = json.dumps(value)
-                    else:
-                        value = str(value)
-                    tags[name] = value
+                properties = model_spec.get("properties", {})
+
+                # convert tags, properties value to string
+                tags = stringify_dictionary(tags)
+                properties = stringify_dictionary(properties)
+
                 tags_to_update = {"replace": tags}
+                properties_to_update = {"add": properties}
```

Comment:
Okay I got this

Reply:
Here we are reading properties from spec file, so we are assuming to add every key:value available in property section of spec file. 
As property supports only addition.We will be only supporting *Add* for property, so if in deployment config see, propertyAsset has only *add* parameter, if there will be replace/delete scenario it will throw exception.
In model_utils we are making sure that no value has been changed, if there is any change we are throwing valid exeption line 181 in model_utils.

Reply:
Thanks for the other comment, resolving it :)